torchtext/data/dataset.py

```diff
@@ -197,6 +197,19 @@ def download(cls, root, check=None):
                             shutil.copyfileobj(gz, uncompressed)
 
         return os.path.join(path, cls.dirname)
+ 
+
+    def filter_examples(self, field_names):
```

Comment:
The argument is `field_names`, while in the code you use `field_name` (it would make sense to allow for multiple fields to be filtered at the same time).

Reply:
Yeah I've looked at the tests and went on to fix them